packages/code/src/renderingPlugin/renderingPlugin.tsx

```diff
@@ -151,6 +154,87 @@ function getHighlightedRowStyle(darkMode: boolean) {
   `;
 }
 
+// Helper functions
+
+/**
+ * Merge consecutive strings into a single string.
+ *
+ * E.g.
+ * ```js
+ * ['_', 'hello ', 'world ', 'hi', {}, 'bye ', '_', 'hello '] => ['_hello world hi', {}, 'bye _hello']
+ * ```
+ */
+const mergeStringsIntoString = (children: Array<string | FlatTokenObject>) => {
+  return children.reduce(
+    (acc: Array<string | FlatTokenObject>, child: string | FlatTokenObject) => {
+      const lastItem = acc[acc.length - 1];
+
+      if (typeof child === 'string') {
+        if (typeof lastItem === 'string') {
+          acc[acc.length - 1] = lastItem + child;
+        } else {
+          acc.push(child);
+        }
+      } else {
+        acc.push(child);
+      }
```

Comment:
I think this can be simplified slightly:


```suggestion
      if (typeof child === 'string' && typeof lastItem === 'string') {
        acc[acc.length - 1] = lastItem + child;
      } else {
        acc.push(child);
      }
```